wifiphisher/common/opmode_manager.py

```diff
@@ -0,0 +1,269 @@
+"""
```

Comment:
We need to follow the naming convention we use across the project. Instead of "opmode_manager.py", I recommend simple "opmode.py"